.npmignore

```diff
@@ -5,7 +5,7 @@
 !/es
 !/lib
 !/src
-!/docs
```

Comment:
I think this should stay as-is. We don't need to publish the docs to npm (that will be done in CI from a git checkout build) and I don't think we want all the additional stuff of which there is a lot...